genvalidity/src/Data/GenValidity.hs

```diff
@@ -804,9 +804,24 @@ instance OVERLAPPING_ GUncheckedSubtermsIncl (K1 i a) b where
   gUncheckedSubtermsIncl (K1 _) = []
 
 
+-- | Generate a valid value by generating all the sub parts,
+-- and trying that until a valid value has been generating
```

Comment:
Maybe adding: "... generating all the sub parts _using `Generic` instance_ ..."

Reply:
Jup, fixed.